webapp/src/codecard.tsx

```diff
@@ -112,7 +112,12 @@ export class CodeCardView extends data.Component<pxt.CodeCard, CodeCardState> {
             {card.time ? <div className="meta">
                 {card.time ? <span key="date" className="date">{pxt.Util.timeSince(card.time)}</span> : null}
             </div> : undefined}
-            {card.extracontent ? <div className="extra content"> {card.extracontent} </div> : undefined}
+            {card.extracontent || card.tags ? <div className="extra content">
+                {card.extracontent}
+                {card.tags ? card.tags.map(tag =>
+                    <span key={`tag${tag.label}`} className={`ui label ${tag.color}`}>{tag.label}</span>
```

Comment:
Can you wrap it in rlf and add a few predefined labels?

Reply:
It is localizable like the rest of our docs. Using md. 
I agree it's not easy, but we don't have localization support for any other md meta data. why block on this?